### ISSUE TITLE
Part cooling fan speed not adjustable when not using Fan 0

### ISSUE THREAD

**Issue by PaulNokel** [dc42/PanelDueFirmware#35]
Hi,

I recently moved my part cooling fan to a pin header on the Duex and assigned it using M563 to the tool (from Fan 0 to Fan 6). I can change/see the speed in DWC. But the PanelDue is always showing 0% and it is not possible to change the speed.

Best regards,
Paul

**Comment by smoki3** [comment]
Yeah same here

**Comment by dc42** [comment]
I suspect you are using old PanelDue firmware. Version 1.22 is current.

**Comment by smoki3** [comment]
@dc42 yes changing the speed is possible. but it is not showing the correct one.

**Comment by wilriker** [comment]
It's planned to improve fan control in version 3.3.
